addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -308,7 +313,7 @@ const prefetchUrls = async (urls = []) => {
             continue;
         }
         try {
-            await processFetchRequest({ request: new Request(url) });
+            await processFetchRequest(new Request(url), true);
```

Comment:
Could we change to `await processFetchRequest(new Request(url), { disableTracking: true });` ?

That way:
- it's much clearer when calling the method that we disable the tracking
- we name the parameter closer to the header name

Reply:
Yes indeed, that's easier to read. It's done.